### DISCUSSION TITLE
Show all images in a post card instead of hiding them behind a carousel

### DISCUSSION THREAD

**Discussion by WEBPerformace** [civitai/civitai#538] [Ideas]
Carousels are UX hell. Scrolling is much easier and more informative. Please show all images in the post card instead of hiding them behind a carousel as this makes them more difficult and frustrating to navigate. I'd rather scroll and quickly view all the images instead of aiming and clicking hundreds of times on all those little arrows.

**Comment by JustMaier** [comment]
This is kind of a tough one.

I agree that navigating carousels on desktop isn't great, being able to go left/right with the keyboard after the first interaction is helpful, but still not ideal, as it requires an additional click.

On the other hand, carousels are helpful for nesting similar content. Generally we use carousels to group things that belong together in a way that makes them take less space (images in a post in the galleries on a model page for example). If we didn't group things like that, then the galleries would easily be flooded by content from just a few users that made large quantity posts.

Have you seen any ways that people have handled this problem that would allow us to still group things?

**Comment by WEBPerformace** [comment]
> On the other hand, carousels are helpful for nesting similar content. Generally we use carousels to group things that belong together in a way that makes them take less space (images in a post in the galleries on a model page for example).

For those who still want to struggle with carousels for this reason, there could be a switch. For example, "show all images in posts" / "group images in posts in a carousel".

If there was such a switch, I would immediately set it to "show all images in posts" for both PC and mobile, and those who prefer carousels for some reason could set it to the second mode, and their UX will not be affected.
